extensions/sql-migration/src/wizard/assessmentDetailsPage/treeComponent.ts

```diff
@@ -0,0 +1,363 @@
+			instanceTableValues = [[
+				{
+					value: this.createIconTextCell(IconPathHelper.sqlServerLogo, this._serverName),
+					style: styleLeft
+				},
+				{
+					value: '0',
+					style: styleRight
+				}
+			]];
+			this._dbNames.forEach((db) => {
+				this._databaseTableValues.push([
+					{
+						value: selectedDbs.includes(db),
+						style: styleLeft
+					},
+					{
+						value: this.createIconTextCell(IconPathHelper.sqlDatabaseLogo, db),
+						style: styleLeft
+					},
+					{
+						value: '0',
+						style: styleRight
+					}
+				]);
+			});
+		} else {
+
+			if (migrationStateModel._targetType === MigrationTargetType.SQLMI && selectedDbs?.length > AZURE_SQL_MI_DB_COUNT_THRESHOLD) {
+				this.wizard.nextButton.enabled = false;
+				this.wizard.message = {
+					level: azdata.window.MessageLevel.Error,
+					text: constants.AZURE_SQL_MI_DB_COUNT_THRESHOLD_EXCEEDS_ERROR(AZURE_SQL_MI_DB_COUNT_THRESHOLD),
+				};
+			}
+
+			instanceTableValues = [[
+				{
+					value: this.createIconTextCell(IconPathHelper.sqlServerLogo, this._serverName),
+					style: styleLeft
+				},
+				{
+					value: migrationStateModel._assessmentResults?.issues?.filter(issue => issue.appliesToMigrationTargetPlatform === migrationStateModel._targetType).length,
+					style: styleRight
+				}
+			]];
+			migrationStateModel._assessmentResults?.databaseAssessments
+				.sort((db1, db2) => db2.issues?.length - db1.issues?.length);
+
+			// Reset the dbName list so that it is in sync with the table
+			this._dbNames = migrationStateModel._assessmentResults?.databaseAssessments.map(da => da.name);
+			migrationStateModel._assessmentResults?.databaseAssessments.forEach((db) => {
+				let selectable = true;
+				if (db.issues.find(issue => issue.databaseRestoreFails && issue.appliesToMigrationTargetPlatform === migrationStateModel._targetType)) {
+					selectable = false;
+				}
+				this._databaseTableValues.push([
+					{
+						value: selectedDbs.includes(db.name) && selectable,
+						style: styleLeft,
+						enabled: selectable
+					},
+					{
+						value: this.createIconTextCell((selectable) ? IconPathHelper.sqlDatabaseLogo : IconPathHelper.sqlDatabaseWarningLogo, db.name),
+						style: styleLeft
+					},
+					{
+						value: db.issues.filter(v => v.appliesToMigrationTargetPlatform === migrationStateModel._targetType)?.length,
+						style: styleRight
+					}
+				]);
+			});
+		}
+		await this._instanceTable.setDataValues(instanceTableValues);
+
+		this._databaseTableValues = selectDatabasesFromList(migrationStateModel._databasesForMigration, this._databaseTableValues);
+		await this._databaseTable.setDataValues(this._databaseTableValues);
+		await this.updateValuesOnSelectionAsync(migrationStateModel);
+		this._databaseCount.value = constants.DATABASES(0, migrationStateModel._databasesForAssessment?.length);
+	}
+
+	private createIconTextCell(icon: IconPath, text: string): string {
```

Comment:
This is define to create a icon and text container. Currently due to a bug in ADS only text can be returned. When the bug is fixed, this will be used to define a container with icon and text. Thus left it there.